src/ui/PaymentDialog/utils.ts

```diff
@@ -99,6 +101,10 @@ function onPaymentSuccess(data) {
     [Tracker.TWQ],
   )
 
+  track.event('Purchase', {
+    category: 'User',
+  })
```

Comment:
Already defined as the `upgrade` event on the [LOC 87](https://github.com/santiment/san-webkit/pull/23/files#diff-c46d8e8ffdf2c976b14c117392bee59eb7d93c57335084df8c17d20c5efdf59eR87)